conda-recipes/scvi/meta.yaml

```diff
@@ -0,0 +1,82 @@
+{% set name = "scvi" %}
+{% set version = "0.1.1" %}
```

Comment:
The version number is 0.1.2 now. We should make  a small change to the `setup.py` so that this version number gets incremented automatically when we type `bumpversion patch`.